__tests__/app.test.js

```diff
@@ -106,8 +84,45 @@ describe('GET commands', () => {
                 ))
             })
         })
-})
+    })
 
+    test('200 status: should return the comments for a given review_id, sorterd in ascending order', () => {
+        return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then(({body})=>{
+            const comments = body.comments;
+            
+            expect(Array.isArray(comments)).toBe(true);
+            expect(comments).toHaveLength(3);
+            comments.forEach((comment)=>{
+                expect(comment).toHaveProperty("comment_id");
+                expect(comment).toHaveProperty("votes");
+                expect(comment).toHaveProperty("created_at");
+                expect(comment).toHaveProperty("author");
+                expect(comment).toHaveProperty("body");
+                expect(comment).toHaveProperty("review_id");
+            });
+            expect(comments).toBeSortedBy("created_at");
+
+        })
```

Comment:
checking that comments is an array isnt a bad test, but it is done indirectly for you when you use .forEach.